### PR TITLE
MAP Fluency Bundle

### PR DESCRIPTION
**Pull request by ryanaguilar**
Earthmover Bundle for NWEA MAP Fluency

Includes refactor for governance and DIIP 7 guidelines

### CODE REVIEW

**Comment by jalvord1** [comment]
hey @ryanaguilar , this looks good - one small thing, can you follow the changes in [this commit](https://github.com/edanalytics/earthmover_edfi_bundles/commit/1c0eb15629a4f3d87d38f83437ea5096fdd97788) and apply to this assessment as well? Essentially anywhere that we are hardcoding ed-fi namespaces in the templates will be replaced with a variable that defaults to Ed-Fi (we need this because SC uses none of the edfi namespaces so we need an easy way to overwrite them)

File: assessments/MAP_Fluency/templates/objectiveAssessments.jsont [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
+  "assessmentItems": [],
+  "assessmentReference": {
+    "assessmentIdentifier": "{{assessment_identifier}}",
+    "namespace": "uri://www.nwea.org/mapFluency/Assessment"
+  },
+  "description": "{{identification_code}}",
+  "identificationCode": "{{identification_code}}",
+  "learningObjectives": [],
```
Comment (jalvord1) [inline assessments/MAP_Fluency/templates/objectiveAssessments.jsont:9]: we can also remove these empty sections (as well as the assessment items above)

File: assessments/MAP_Fluency/seeds/academicSubjectDescriptors.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+assessmentFamily,academicSubjectDescriptor
+NWEA MAP Fluency,uri://ed-fi.org/AcademicSubjectDescriptor#Reading
```
Comment (jalvord1) [inline assessments/MAP_Fluency/seeds/academicSubjectDescriptors.csv:2]: this can just be included in the assessments.csv!

**Comment by rlittle08** [comment]
@jalvord1 testing on 2024 and 2025 files in boston dev is done and looks good 👍

**Review by jalvord1 — APPROVED** [review]

**Pull request merged**